arcade/paths.py

```diff
@@ -47,6 +71,16 @@ def _heuristic(start: Point, goal: Point):
 
 class _AStarGraph(object):
     # Define a class board like grid with two barriers
+    """
+    A gird with barriers that is used in _AStarSearch
```

Comment:
```suggestion
    A grid which tracks barriers to movement.
```
There's a typo (s/gird/grid) in this line. I also think "that is used in _AStarSearch" is redundant here because the name of the class communicates the idea.